app/models/user.rb

```diff
@@ -16,7 +23,7 @@ def assign_from_auth_hash(auth_hash)
     github_attributes = {
       github_id: auth_hash['uid'],
       github_login: auth_hash['info']['nickname'],
-      access_token: auth_hash.dig('credentials', 'token')
+      access_token: auth_hash.dig('credentials', 'token'),
```

Comment:
I know some people prefer to have to have a trailing commas to make the diff cleaner, but my inner rubocop is sad.

What do you think about undoing this change?